cmd/buildah/mkcw.go

```diff
@@ -0,0 +1,104 @@
+		Args:    cobra.ExactArgs(2),
+	}
+	mkcwCommand.SetUsageTemplate(UsageTemplate())
+	rootCmd.AddCommand(mkcwCommand)
+	flags := mkcwCommand.Flags()
+	flags.SetInterspersed(false)
+
+	flags.StringVarP(&teeType, "type", "t", "", "TEE type")
+	flags.StringVarP(&options.AttestationURL, "attestation-url", "u", "", "attestation server URL")
+	flags.StringVarP(&options.AttestationURL, "attestation_url", "", "", "attestation server URL (alternate flag spelling)")
+	if err := flags.MarkHidden("attestation_url"); err != nil {
+		panic("error marking attestation_url as hidden")
+	}
+	flags.StringVarP(&options.BaseImage, "base-image", "b", "", "alternate base image (default: scratch)")
+	flags.StringVarP(&options.BaseImage, "base_image", "", "", "alternate base image (default: scratch) (alternate flag spelling)")
+	if err := flags.MarkHidden("base_image"); err != nil {
+		panic("error marking base_image as hidden")
+	}
+	flags.StringVarP(&options.DiskEncryptionPassphrase, "encryption-passphrase", "p", "", "disk encryption passphrase")
+	flags.StringVarP(&options.DiskEncryptionPassphrase, "encryption_passphrase", "", "", "disk encryption passphrase (alternate flag spelling)")
+	if err := flags.MarkHidden("encryption_passphrase"); err != nil {
+		panic("error marking encryption_passphrase as hidden")
+	}
+	flags.IntVarP(&options.CPUs, "cpus", "c", 0, "number of CPUs to expect")
+	flags.IntVarP(&options.Memory, "memory", "m", 0, "amount of memory to expect (MB)")
+	flags.StringVarP(&options.WorkloadID, "workload-id", "w", "", "workload ID")
+	flags.StringVarP(&options.WorkloadID, "workload_id", "", "", "workload ID (alternate flag spelling)")
+	if err := flags.MarkHidden("workload_id"); err != nil {
+		panic("error marking workload_id as hidden")
+	}
+	flags.StringVarP(&options.Slop, "slop", "s", "25%", "extra space needed for converting a container rootfs to a disk image")
+	flags.StringVarP(&options.FirmwareLibrary, "firmware-library", "f", "", "location of libkrunfw-sev.so")
+	flags.StringVarP(&options.FirmwareLibrary, "firmware_library", "", "", "location of libkrunfw-sev.so (alternate flag spelling)")
+	if err := flags.MarkHidden("firmware_library"); err != nil {
+		panic("error marking firmware_library as hidden")
+	}
+	flags.BoolVarP(&options.IgnoreAttestationErrors, "ignore-attestation-errors", "", false, "ignore attestation errors")
+	flags.BoolVarP(&options.IgnoreAttestationErrors, "ignore_attestation_errors", "", false, "ignore attestation errors (alternate flag spelling)")
+	if err := flags.MarkHidden("ignore_attestation_errors"); err != nil {
+		panic("error marking ignore_attestation_errors as hidden")
+	}
+	flags.BoolVarP(&options.IgnoreChainRetrievalErrors, "ignore-chain-retrieval-errors", "", false, "ignore errors retrieving the certificate chain")
+	flags.BoolVarP(&options.IgnoreChainRetrievalErrors, "ignore_chain_retrieval_errors", "", false, "ignore errors retrieving the certificate chain (alternate flag spelling)")
```

Comment:
When would I want one versus the other? ignore_attestation_errors versus ignore_chain_retrieval_errors?

Reply:
It's mainly about separating local errors (missing tools, missing firmware, permissions) from remote errors (server down, authentication errors).  I guess we can just call them all attestation errors.

Reply:
Who would use these options?  If the mkcw command is not going to succeed, why would I want to ignore errors?  And why would I need to flags to ignore specific errors?

Reply:
Maybe it'd be undocumented, but we depend on it in order to be able to verify that the image looks like we expect it to look in our tests, to the extent that we can when we don't have the hardware.

Reply:
Should we hide the option then?

Reply:
Sure.